premake/premake5.crashpad.lua

```diff
@@ -8,14 +8,14 @@ MACOS_SYSROOT = '/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.pla
 function crashpad_common()
   language "C++"
   cppdialect "C++14"
+	symbols "On"
```

Comment:
Can we use spaces in .lua files, please?

Reply:
indeed